### DISCUSSION TITLE
Using a lora device as a raido

### DISCUSSION THREAD

**Discussion by W-M-D** [iNavFlight/inav#8714] [General]
Hello, I'm planning on using a meshtastic lora device as a radio, so it can pick up special commands tagged with "DRONE" and send them to the flight controller via uart. Am i crazy? This way the drone can not only accept commands but also relay other packets for the mesh.

**Comment by rotorHead82** [comment]
You are not crazy. I am thinking this as well. I will be carrying a Mesthastic Node in my plane as a secondary/recovery tracker. I have my own Mesthastic network running in the aera that I fly with great aerial coverage.
My thought is to use the Serial Module of the Meshtastic node to talk to the FC. Sending commands to the FC through the meshtastic node could trigger actions like RTH etc.
Something like this is already implemented in INav using a SMS GSM module. For the Meshtastic node it would be even simpler as we dont need all the AT commands to be sent. A simple string with the telemetry data from the FC through the UART could be sent as a simple message through the Mesthtastic network.
Someone with a bit more programming skills than me shoul be able to implement this fairly easily.
